addons/web/static/tests/views/fields/daterange_field_tests.js

```diff
@@ -588,4 +591,67 @@ QUnit.module("Fields", (hooks) => {
             "the end date should only show date when option formatType is Date"
         );
     });
+
+    QUnit.test("when same locales have differences (moment vs luxon)", async function (assert) {
+        patchWithCleanup(localization, { dateTimeFormat: "dd MMM, yyyy HH:mm:ss" });
+
+        // Patch the moment locale into a pseudo locale that will get different from the luxon one
+        const originalLocaleName = moment.locale();
+        const originalLocaleData = moment.localeData();
+        moment.updateLocale(originalLocaleName, {
+            monthsShort: [
+                "JaBon",
+                "FeBob",
+                "MaBor",
+                "ApBor",
+                "MaBoy",
+                "JuBon",
+                "JuBol",
+                "AuBog",
+                "SeBop",
+                "OcBot",
+                "NoBov",
+                "DeBoc",
+            ],
+        });
```

Comment:
I don't know what that's supposed to do, but apparently it's incorrectly cleaned up because it breaks the gridview tests.

Reply:
Oh snap, I failed polluting the world without being noticed 😅